recipe/meta.yaml

```diff
@@ -73,6 +73,8 @@ requirements:
 
 test:
   requires:
+    # The test requires objdump which is part of the binutils package.
+    # Maybe add this as a test dependency on all platforms?
     - binutils      [linux and aarch64]
```

Comment:
yeah, possibly on all unix architectures. But as it doesn't fail on other target so far, I am fine as is